proto/zitadel/settings/object/v3alpha/object.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
```

Comment:
The structure of the object folders break my mind. as they look exactly the same why do we not use the same object?

Reply:
This specific object is replaced by the resources object in #8357
The v3 objects won't look the same anymore.